### PR TITLE
feat(CR-UI): Added Advaced filter for CR & fix # of components count

### PR DESCRIPTION
**Pull request by akapti**
[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

> Please provide a summary of your changes here.
 * Added `Advanced Filter` for `Clearing Request`, based on `Priority`, `BA/BL` and `Status`.
 * Did you add or update any new dependencies that are required for your change? - **No**

Issue: #1036 

### How To Test?
* Test the advanced filter are for CR are working fine.
* Check the `# of components to be cleared` shows the number of releases in with clearing 
* Have you implemented any additional tests? - **No**

### Checklist
Must:
- [x] All related issues are referenced in commit messages and in PR

Signed-off-by: Abdul Kapti <abdul.mannankapti@siemens.com>

### CODE REVIEW

**Comment by akapti** [comment]
Screenshot:
image

**Comment by mcjaeger** [comment]
(from dicussion) two proposals:

* Progress with capital P
* pls consider renaming business line with group

**Review by smrutis1 — CHANGES_REQUESTED** [review]
Reviewed and tested:

* It's working fine after adding custom field(BA BL) in control panel.

* It's not working without adding `custom field 'BA BL'` in control panel.
image

File: frontend/sw360-portlet/src/main/resources/META-INF/resources/html/moderation/view.jsp [hunk 1]
```diff
@@ -272,12 +322,29 @@ AUI().use('liferay-portlet-url', function () {
...
-                $("#date_type").val("");
-                $('#date_range').hide().val("");
+                $('.cr_filter').val("");
                 if (tab === '#tab-OpenCR') {
                     $("#date_type option[value="+"<%=ClearingRequest._Fields.TIMESTAMP_OF_DECISION%>"+"]").hide().attr("disabled", "");
+                    $("#cr_priority_div").show();
+                    $('#cr_status option').each(function() {
+                        let val = $(this).attr("value");
+                        if (val === "2" || val === "5") {
+                            $(this).hide().attr("disabled", "");
```
[the remark this answers is not in the document]
Comment (akapti) [reply frontend/sw360-portlet/src/main/resources/META-INF/resources/html/moderation/view.jsp]: Changes done.

**Comment by akapti** [comment]
> Reviewed and tested:
It's not working without adding custom field 'BA BL' in control panel.

Fixed the above mentioned issue.

**Review by smrutis1 — APPROVED** [review]
Code looks good. Tested successfully.

**Pull request merged**